Assets/General/Scripts/ScreenText/TextAndTimerData.cs

```diff
@@ -12,7 +12,7 @@
 /// <summary>
 /// The data class for text and time
 /// </summary>
-[Serializable]
+/*[Serializable]
```

Comment:
same here - delete unused code we can retrieve it later if needed bc version control is cool like that